### ISSUE TITLE
Memory footprint not available

### ISSUE THREAD

**Issue by lapo-luchini** [gliaskos/freebsd-chromium#155]
In the internal _Task Manager_ the **Memory footprint** column doesn't show:
![image](https://user-images.githubusercontent.com/420454/59747122-882ea580-9278-11e9-8786-527083df3c65.png)

**Comment by lapo-luchini** [comment]
Something has changed in `76.0.3809.132`: now it shows `0K` instead of `-`. :)
![image](https://user-images.githubusercontent.com/420454/65697519-5bf6a200-e07b-11e9-9884-737a1b37bf06.png)

**Comment by mwolf-rw** [comment]
Yes, the feature is partially implemented. Basically the stuff reading the RSS memory size is missing currently IIRC.
